src/main/java/io/github/darealturtywurty/turtychemistry/common/TurtyTags.java

```diff
@@ -0,0 +1,14 @@
+package io.github.darealturtywurty.turtychemistry.common;
+
+import io.github.darealturtywurty.turtychemistry.TurtyChemistry;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.tags.BlockTags;
+import net.minecraft.tags.ItemTags;
+import net.minecraft.tags.TagKey;
+import net.minecraft.world.item.Item;
+import net.minecraft.world.level.block.Block;
+
+public record TurtyTags() {
```

Comment:
I don't really see why this is a record. That is pointless. Just keep it as a class like everything else.